docs/docs/playground/components/SignTransactionDemo.tsx

```diff
@@ -13,7 +13,9 @@ export const SignTransactionDemo = () => {
     let error = "";
 
     try {
-      signedTransaction = await signTransaction(transactionXdr);
+      signedTransaction = await signTransaction({
+        transactionXdr,
+      });
```

Comment:
same - there shouldnt be any changes here

Reply:
Hmm with `signedTransaction = await signTransaction(transactionXdr);` (how it was before), there is no popup happening. I get `[object object]`. Also, our signTransaction documentation says do `signTransaction({ transactionXdr: string })`.

Reply:
@piyalbasu I realized I need to update lyra api to `1.0.0-alpha.7` in package.json 🤓 . Will make an update.